macros config meta and docs (#8790)
fishtown-analytics/docs.getdbt.com: website/docs/reference/macro-properties.md

import PropsCallout from '/snippets/_config-prop-callout.md';

> You can declare macro properties in any `properties.yml` file. <PropsCallout title={frontMatter.title}/> 

Reviewer: Meta is not a property anymore, it's a config. We should update this language

Author: Updated [doc](https://github.com/dbt-labs/docs.getdbt.com/pull/8790/changes#diff-258721792aef76b6269d663aa2304abe84fe9c796672c06032da5976baecad60), preview [here](https://docs-getdbt-com-git-macros-config-dbt-labs.vercel.app/reference/macro-properties)

> You can declare macro properties in any `properties.yml` file. <PropsCallout title={frontMatter.title}/> 

Author: ```suggestion
You can declare macro properties and configs in any `properties.yml` file. <PropsCallout title={frontMatter.title}/> 
```

> You can declare macro properties and configs in any `properties.yml` file. <PropsCallout title={frontMatter.title}/> 

Author: ```suggestion
You can declare macro properties and configs in any `.yml` file in your project. <PropsCallout title={frontMatter.title}/> 
```

> You can declare macro properties and configs `.yml` files in your project. <PropsCallout title={frontMatter.title}/> 

Author: ```suggestion
You can declare macro properties and configs in `.yml` files in your project. <PropsCallout title={frontMatter.title}/> 
```

> You can declare macro properties and configs in any `.yml` file in your project. <PropsCallout title={frontMatter.title}/> 

Author: ```suggestion
You can declare macro properties and configs `.yml` files in your project. <PropsCallout title={frontMatter.title}/> 
```
